js/src/common/compat.js

```diff
@@ -92,7 +91,6 @@ export default {
   'utils/stringToColor': stringToColor,
   'utils/Stream': Stream,
   'utils/subclassOf': subclassOf,
-  'utils/SuperTextarea': SuperTextarea,
```

Comment:
Should we provide BasicEditorDriver as a (deprecated) alias for this? All the logic from there is still there, but we'd need to include something from forum in common. Although in this case I don't think it's too bad since BasicEditorDriver doesn't import anything from the rest of forum.

Reply:
Since the logic is the same, and there's always a chance an extension could've used it, I don't see why not.

Interesting that it was in common though since it was only used in forum.

Reply:
This actually brings up an interesting point: it was in common, not forum, because it didn't depend on any stuff in forum. And that might be something we want to keep: TextEditor might be eventually useful for editing stuff on the admin side that uses formatting (e.g. if tags descriptions adopt formatting).

For now why don't we just add it to forum, but we can keep an eye out for this in the future?

Reply:
Agreed.